bench/run.ml

```diff
@@ -3,19 +3,24 @@ open Bos
 
 let ( <.> ) f g = fun x -> f (g x)
 
-let run max =
+let run max json =
+  let json = match json with true -> "-j" | false -> "" in
   let open OS.Cmd in
   (out_run_in <.> run_out) Cmd.(v "cat" % "/dev/urandom")
   >>= (out_run_in <.> run_io Cmd.(v "./bench/zpipe"))
-  >>= (R.ok <.> run_io ~err:err_null Cmd.(v "./bench/bench.exe" % string_of_int max))
+  >>= ( R.ok
+      <.> run_io ~err:err_null
+            Cmd.(v "./bench/bench.exe" % string_of_int max % json) )
```

Comment:
You missed the `-m` option according to: https://github.com/mirage/decompress/pull/95/files#diff-27f0ee83ca8d56480dc38863cf1eba96171396ee92d0e5ae25d35c914085e7dfR100-R102